### ISSUE TITLE
Adjacent assignments aligned analysis statement wrong

### ISSUE THREAD

**Issue by ravage84** [squizlabs/PHP_CodeSniffer#1036]
May be I have understood something wrong, but I think there is an error or at least a wrong statement in the "Analysis of Coding Conventions for CakePHP" and thus probably in all such analyses.

http://squizlabs.github.io/PHP_CodeSniffer/analysis/cakephp/cakephp/index.html#adjacent-assignments-aligned

> For a block of assignments, are the assignment operators (equals, plus-equals etc.) aligned?
> yes 	82.12%
> no 	17.88%

I wasn't sure If I understood the statement correctly, so I tested it with some dummy code.

``vendor\bin\phpcs.bat --standard=Generic --report=info -p -v --sniffs=Generic.Formatting.MultipleStatementAlignment src/test.php``

```` php
<?php
$var1           = 'value';
$longerVariable = 'bla';
?>
````

````
PHP CODE SNIFFER INFORMATION REPORT
----------------------------------------------------------------------
Adjacent assignments aligned: yes [1/1, 100%]
````

```` php
<?php
$var1 = 'value';
$longerVariable = 'bla';
?>
````

````
PHP CODE SNIFFER INFORMATION REPORT
----------------------------------------------------------------------
Adjacent assignments aligned: no [1/1, 100%]
````

So this would mean that the CakePHP framework does align multiple, adjacent statements with spaces.
If that is the intended statement, then this would be wrong as the CakePHP framework does NOT align them.

That would mean, as mentioned above, that the compiled statistiscs for all analysed code is wrong, too:
http://squizlabs.github.io/PHP_CodeSniffer/analysis/index.html#adjacent-assignments-aligned

**Comment by gsherwood** [comment]
Yep, I think you're right. The metrics shouldn't be recorded if a single assignment is found, but it is doing that now because it is also checking that single assignments have a single space before them. That's great for coding standards, but not so great for metrics.

I'll change the sniff and do another run to regenerate the reports. Thanks for finding and reporting this.

**Comment by gsherwood** [comment]
New run data has been generated and the results look correct now. Thanks again.
